shared/output/cloud/showcase/lib/google-showcase.rb

```diff
@@ -0,0 +1 @@
+require "google/showcase"
```

Comment:
Do you think we should do showcase with the cloud or base generator? I think it may be better to use the base templates for showcase and real APIs for the cloud generator. WDYT?

(Whatever we decide let's leave this the way it is for this PR)

Reply:
The acceptance tests run off of the default generator. The fact that we are still testing showcase on the cloud generator is an artifact to back when cloud was the only working generator and default was using a different implementation. Pre #109.

Reply:
Hm, I'm still confused. If it's running off of the default generator then this extra file shouldn't exist? I see the `Rakefile` has a TODO for the template set so is something still hardwired or why does this file get generated?

Reply:
The acceptance tests run the generator when the tests are run, and don't use these files. These files are used to test the cloud generator output to ensure it is working as expected.

There are features that right now only showcase uses, and in order to get the broadest test coverage we were testing the cloud generator against the showcase protobufs.

Reply:
Ah, I see - thanks for clarifying.